### ISSUE TITLE
results on davis

### ISSUE THREAD

**Issue by qchenclaire** [foolwood/SiamMask#11]
Did you also train SiamMask using random pairs of the same sequence from DAVIS dataset? I'm just curious because DAVIS has much much more occlusions than VOT.  I doubt if siamese network works well on DAVIS. It's too difficult if the template is one part of the object and the target is another part of the object.

**Comment by lzrobots** [comment]
@qchenclaire

**Comment by lzrobots** [comment]
@qchenclaire 1. We didn’t use training data from DAVIS. As stated in the paper, the datasets we used for training are from COCO, DET, VID and Youtube-VOS. 

2. On the contrary,  we think DAVIS has **fewer** occlusions  and much shorter sequence (~50 vs ~500 frames per video) than VOT.  

3. As SiamMask is trained with instance object mask so I would say it cannot do very well on DAVIS2016 as there are multiple objects present but with single label in DAVIS2016. For example, SiamMask intends to output the instance mask of either person or pushchair in a video but when these two exist in a video with a single class label, SiamMask produces poor performance. Fortunately we have DAVIS2017 challenge that different object have different label in a video. So SiamMask can do well on DAVIS2017.  

4. Online finetuning methods always suffer at this point as they only remember the appearance of the first frame and cannot adapt to out-of-plane rotation very well. However, thanks to the object mask training, SiamMask gains a better instance-level understanding of the object. Fine-tuning could also be a compliment to our method. We will later show it can improve our performance on VOS.
